scripts/e2e-test.sh

```diff
@@ -3,8 +3,6 @@ SRC_PATH=$1
 CUSTOM_COMMAND="${2:-yarn test}"
 GATSBY_PATH="${CIRCLE_WORKING_DIRECTORY:-../../}"
 
-npm install -g gatsby-dev-cli &&
```

Comment:
I do think it's nicer to have this re-usable e2e-test utility and have it not _require_ additional dependencies. What do you think?

Reply:
sure, I will bring it back